eclcli/common/utils.py

```
@@ -347,6 +347,18 @@ def validate_ipv4(text):
         raise exceptions.CommandError(msg % text)
     return text
 
+def validate_int_range(text, min_val, max_val):
+    try:
+        int_text = int(text)
+
+        if min_val <= int_text <= max_val:
+            return int_text
+    except ValueError:
+        pass
```

You should raise the exception with not int message here.